components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/util/designview/beans/configs/siddhielements/sourcesink/SourceSinkConfig.java

```diff
@@ -32,17 +32,18 @@
     private String type;
     private List<String> options;
     private MapperConfig map;
+    private boolean correlateIdExist;
```

Comment:
A better name would be isCorrelationIdExists.

Reply:
fixed